src/sandbox/generate-icons.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const path = require('path');
+var getDirName = require('path').dirname;
+
+const pathToUsaIcons = '../../node_modules/uswds/dist/img/usa-icons';
+
+function readIcons(dir) {
+  const files = {};
+
+  fs.readdirSync(path.join(__dirname, dir)).forEach(filename => {
+    const data = fs.readFileSync(path.join(__dirname, dir, filename), {encoding: 'utf-8'});
+    files[`src/img/usa-icons/${filename}`] = data;
+  });
+
+  
+  writeFile(path.join(__dirname, 'icons.json'), JSON.stringify(files), () => {});
+
+  return files;
+}
+
+function writeFile(path, contents, cb) {
+  fs.mkdir(getDirName(path), { recursive: true}, function (err) {
+    if (err) return cb(err);
+
+    fs.writeFile(path, contents, cb);
+  });
+}
+
+readIcons(pathToUsaIcons);
```

Comment:
Main chunk of PR here - this script will run whenever `npm run start` is called, and it creates an `icons.json` file in `src/sandbox/` directory containing key value mapping of all usa-icons in uswds' image directory. The key is each svg's file name, and the value is the svg template as a string. This is then referenced when setting up dependencies in sandbox, and allows us to add the image folder in sandbox environment